scalafix-tests/src/test/resources/checkSyntax/NoXml.source

```diff
@@ -0,0 +1,99 @@
+rewrites = [NoXml]
+
+<<< don't add import when not needed
+object A {}
+>>>
+object A {}
+<<< single line, no splice
+object A {
+  <div></div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  xml"""<div></div>"""
+}
+
+<<< single line, splice
+object A {
+  val bar = "bar"
+  <div>{bar}</div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  val bar = "bar"
+  xml"""<div>${bar}</div>"""
+}
+
+<<< multi-line, no splice
+object A {
+  val foo =
+    <div>
+      <span>Hello</span>
+    </div>
+}
+>>>
+import scala.xml.quote._
+object A {
+  val foo =
+    xml"""<div>
```

Comment:
The whitespaces in the start/end don't get included. Bu otherwise all the whitespace are preserved. So we should not shift right/left the xml literal.